nextcord/abc.py

```diff
@@ -1366,10 +1366,20 @@ async def send(
         if embed is not None and embeds is not None:
             raise InvalidArgument("Cannot pass both embed and embeds parameter to send()")
 
+        local_embed_files = []
         if embed is not None:
+            if embed._local_image:
+                local_embed_files.append(embed._local_image)
+            if embed._local_thumbnail:
+                local_embed_files.append(embed._local_thumbnail)
+
             embed = embed.to_dict()
 
         elif embeds is not None:
+            local_embed_files.extend([e._local_image for e in embeds if bool(e._local_image)])
+            local_embed_files.extend(
+                [e._local_thumbnail for e in embeds if bool(e._local_thumbnail)]
+            )
```

Comment:
Is the cast to bool necessary?
```suggestion
            local_embed_files.extend([e._local_image for e in embeds if e._local_image])
            local_embed_files.extend(
                [e._local_thumbnail for e in embeds if e._local_thumbnail]
            )
```